docs/adding_a_new_gravity_endpoint_into_metaphysics.md

```diff
@@ -0,0 +1,78 @@
+## Adding a gravity endpoint into Metaphysics
+
+This doc is aimed at helping backend devs or beginner front end devs on how to add an existing gravity endpoint into Metaphysics.
+
+This article assumes that you already have your local machine setup and have your graphiql.app or other app (Postman, Insomnia etc) setup. If you need help with that, you can start [here](https://github.com/artsy/metaphysics#setting-up-your-local-graphiql)
+
+### Steps to add an existing endpoint
+
+1. Find an existing PR or commit which adds something similar to what you want. If you are looking for a PUT endpoint, then [this](https://github.com/artsy/metaphysics/commit/987b21f501bb3483d196720d4a252c25b704352e) is a great example.
```

Comment:
It might be a bit better to link directly to a PR, versus a commit

Reply:
Updated in the new branch. Will make a PR soon and link it back here